Mathlib/SetTheory/Ordinal/NaturalOps.lean

```diff
@@ -497,17 +496,11 @@ theorem zero_nmul (a) : 0 ⨳ a = 0 := by rw [nmul_comm, nmul_zero]
 @[simp]
 theorem nmul_one (a : Ordinal) : a ⨳ 1 = a := by
   rw [nmul]
-  simp only [lt_one_iff_zero, forall_eq, nmul_zero, nadd_zero]
-  convert csInf_Ici (α := Ordinal)
+  convert csInf_Ici
   ext b
-  -- Porting note: added this `simp` line, as the result from `convert`
-  -- is slightly different.
-  simp only [Set.mem_setOf_eq, Set.mem_Ici]
-  refine ⟨fun H => le_of_forall_lt fun c hc => ?_, fun ha c hc => ?_⟩
-  -- Porting note: had to add arguments to `nmul_one` in the next two lines
-  -- for the termination checker.
-  · simpa only [nmul_one c] using H c hc
-  · simpa only [nmul_one c] using hc.trans_le ha
+  refine ⟨fun H ↦ le_of_forall_lt (a := a) fun c hc ↦ ?_, fun ha c hc ↦ ?_⟩
+  · simpa [nmul_one c] using H c hc
+  · simpa [nmul_one c] using hc.trans_le ha
```

Comment:
We have just `nmul_one` here? Otherwise I think the porting note still applies.
```suggestion
  · simpa [nmul_one c] using H c hc
  · simpa [nmul_one c] using hc.trans_le ha
```